scripts/release_config/base_digests

```diff
@@ -0,0 +1,2 @@
+amd64=sha256:48a4bec3d1dec90b5dd5420bf7c41a5756b7fbe8b862546134fbe2caa607679f
+arm64=sha256:ad1de47449386ae0f6009c38d714cbcc218d6f8087ea2d45272b671baa5072af
```

Comment:
These are the arch-specific digests of images from the `registry.access.redhat.com/ubi8/ubi-minimal:8.4`.  They are used to explicitly cross-compile the binaries to their respective image archs.

Reply:
I've changed the name to `rhel-ubi-minimal-8-4-arch-digests` to make it more clear.

Reply:
It'd be good to document this in a comment. Maybe including steps to reproduce, something like:
```
podman pull --platform=linux/arm64 ubi-minimal:8.4
podman inspect ubi-minimal:8.4 | jq -r '.[]["RepoDigests"][0] | sub(".+?(?=sha)";"")'
```

Also, AFAIK ubi-minimal:8.4 tag isn't pinned but gets updated with new z-streams, so Q would be whether we need to pin the arch specific images nevertheless?

Reply:
My thought here was to pin the arch-images to known compatible digests from the same ubi-minimal:8.4 release. This is primarily to keep all releases identical unless otherwise deliberately updated (which could then be documented in the release body). 

I agree WRT commenting the fetch process.